client/src/pages/ProfileListings/ProfileListings.tsx

```diff
@@ -6,13 +6,30 @@ import SearchIcon from '@material-ui/icons/Search';
 import DateRangeIcon from '@material-ui/icons/DateRange';
 import ProfilePreview from './ProfilePreview/ProfilePreview';
 import generateMockData from './generateMockData';
+import { defaultUsers } from './mockData';
 import useStyles from './useStyles';
 
 const ProfileListings = (): JSX.Element => {
   const classes = useStyles();
   const [currentUsers, setCurrentUsers] = useState<number>(6);
   const [mockData, setMockData] = useState<any>([]);
 
+  const defaultPreviews = defaultUsers.map((el, idx) => {
+    return (
+      <Grid item key={idx}>
```

Comment:
Using indexes for keys is an anti-pattern in React and should be avoided.

Reply:
The default preview will only be rendered at start as a placeholder until the data from the api is retrieved. I know the key prop is used by react to determine what to re-render and not, but does it matter for a component that is just used as a placeholder on launch? @rajivtitus What would you use as a key prop in this instance?

Reply:
@ababber Maybe you can consider using a loading spinner rather than showing a preview of mock profiles.

Using indexes as keys is fine if the list is static. However, when we have dynamic lists we need to use unique keys (not just indexes)

Reply:
I went with a loading spinner. Thanks for the suggestion!

Reply:
I used circular progress instead of loading a static placeholder. Thanks for the suggestion!